lib/rom/sql/commands_ext/postgresql.rb

```diff
@@ -0,0 +1,19 @@
+module ROM
+  module SQL
+    module Commands
+      module PostgreSQL
+        class Create < ROM::Commands::Create
+          def execute(tuples)
+            Array([tuples]).flatten.map do |tuple|
+              attributes = input[tuple]
+              validator.call(attributes)
+              relation.returning(*attributes.keys).insert(attributes.to_h).first
+            end
```

Comment:
We should have something like `Create#insert` that can be overridden and simply implement it here as:

``` ruby
def insert(attributes)
  relation.returning(*attribute.keys).insert(attributes.to_h).first
end
```

With the default implementation that has the original behavior of inserting then fetching the row again using pk.

WDYT?